app/views/admin/merchants/index.html.erb

```diff
@@ -1,4 +1,4 @@
 <%= render partial: 'admin/header' %>
 <% @merchants.each do |merchant| %>
-<p><%= merchant.name %></p>
+<p><%= link_to "#{merchant.name}", admin_merchant_path(merchant.id) %></p>
```

Comment:
I think Brian said we don't need to include `.id` when we use ruby paths I think we can just call `admin_merchant_path(merchant)`, and ruby knows what to pull from that object. But we can play around with that on the next story if we want or you can try it before we pull since there are merge conflicts on the PR ahead of this one.